src/ripplegw/waveforms/NRTidalv3_utils.py

```
@@ -96,14 +96,16 @@ def _get_merger_frequency(theta: Array):
 
-    # convert from angular frequency to frequency (divide by 2*pi) and then convert from dimensionless frequency to Hz (divide by mtot * LAL_MTSUN_SI)
+    # convert from angular frequency to frequency (divide by TWO_PI) and then convert from dimensionless frequency to Hz (divide by M * MTSUN)
     fHz_merger = Momega_merger / TWO_PI / (M * MTSUN)
 
     return fHz_merger
 
 
 # Full tidal correction
-def fullTidalPhaseCorrection(Mf: Array, theta_intrinsic: Array, P_P: Array):
+def fullTidalPhaseCorrection(
+    Mf: Array, theta_intrinsic: Array, P_P: Float[Array, " n_freq"]
```

Ruff (0.15.9)

[error] Syntax error in forward annotation: Expected an expression

(F722)